docs/source/markdown/podman-network-create.1.md

```diff
@@ -24,6 +24,11 @@ release because it is used as a special network mode in **podman run/create --ne
 Disables the DNS plugin for this network which if enabled, can perform container to container name
 resolution.
 
+#### **--dns**
```

Comment:
this accepts a value so you need to add `=*ip*`

also mention that this can be set several times